specification/storage/resource-manager/Microsoft.Storage/stable/2023-05-01/storageTaskAssignments.json

```diff
@@ -472,6 +472,7 @@
           "enum": [
             "ValidateSubscriptionQuotaBegin",
             "ValidateSubscriptionQuotaEnd",
+            "Accepted",
```

Comment:
Is this change a result of migrating to RPaaS?
For enums, setting `modelAsString` to `true` is recommended. This approach allows for greater flexibility in adding new values without causing breaking changes.

Reply:
Yes, this is a result of migrating to RPaaS. That setting would have been ideal, but at this point, ARM Review Board said changing this to true will be too breaking.